Tools/Scripts/libraries/webkitscmpy/webkitscmpy/mocks/local/git.py

```diff
@@ -1065,6 +1065,100 @@ def reset(self, index):
         self.head = self.commits[self.head.branch][-(index + 1)]
         return mocks.ProcessCompletion(returncode=0)
 
+    def resolve_all_commits(self, branch, remote=None):
+        if not remote:
+            all_commits = self.commits[branch].copy()
+        else:
+            all_commits = self.remotes['{}/{}'.format(remote, branch)].copy()
+        last_commit = all_commits[0]
+        while last_commit.branch != branch:
+            head_index = None
+            if not remote:
+                commits_part = self.commits[last_commit.branch]
+            else:
+                commits_part = self.remotes['{}/{}'.format(remote, last_commit.branch)]
+            for i in range(len(commits_part)):
+                if commits_part[i].hash == last_commit.hash:
+                    head_index = i
+                    break
+            all_commits = commits_part[:head_index] + all_commits
+            last_commit = all_commits[0]
+            if last_commit.branch == self.default_branch and last_commit.identifier == 1:
+                break
+        if remote:
+            for commit in all_commits:
+                setattr(commit, '__mock__remotes', set([remote]))
+        return all_commits
+
+    def rev_list(self, something):
```

Comment:
Is this the replacement of `commits_in_range`?

Reply:
Correct, Ditto.